tools/sidekick/config.json

```diff
@@ -23,15 +23,15 @@
       "url": "https://main--milo--adobecom.hlx.page/tools/loc/index.html?project=cc--adobecom",
       "passReferrer": true,
       "includePaths": [ "**/:x**" ]
-    },
-    
+    },    
     {
       "containerId": "tools",
       "id": "floodgate",
       "title": "Floodgate",
       "environments": [ "edit" ],
-      "url": "https://main--milo--adobecom.hlx.page/tools/floodgate/index.html?project=cc--adobecom",
+      "url": "https://main--milo--adobecom.hlx.page/tools/floodgate?milolibs=floodgateui",
```

Comment:
Hi @sukamat , I see error when trying to access the url 
<img width="1905" alt="Screen Shot 2023-11-02 at 9 58 42 AM" src="https://github.com/adobecom/cc/assets/69535463/698a5674-a060-42c1-9bbf-a2fe1648e33a">